linkerd/config/src/test/scala/io/buoyant/linkerd/config/ParserTest.scala

```diff
@@ -0,0 +1,143 @@
+package io.buoyant.linkerd.config
+
+import com.fasterxml.jackson.core.JsonParseException
+import com.twitter.finagle.Dtab
+import com.twitter.util.{Return, Throw, Try}
+import io.buoyant.linkerd.config.http._
+import io.buoyant.linkerd.config.namers.FileSystemNamerConfig
+import io.buoyant.linkerd.config.thrift._
+import org.scalatest.FunSuite
+
+
+class ParserTest extends FunSuite {
+  val YamlConfig = """
+baseDtab: /foo => /bar ;
+
+namers:
+  - kind: io.l5d.fs
+    rootDir: /tmp
+
+routers:
+  - protocol: http
+    httpUriInDst: true
+
+  - protocol: http
+    label: httpExt
+    servers:
+      - port: 8080
+        ip: 0.0.0.0
+    timeoutMs: 1000
+    dstPrefix: /ext/http
+
+  - protocol: thrift
+    thriftFramed: false
+    thriftMethodInDst: true
+    baseDtab: |
+      /thrift => /$/io.buoyant.fs/thrift;
+"""
+  val configs: ParseResult = Parser(YamlConfig)
+  def baseConfig = configs.parsedConfig.get
+  def validatedConfig = configs.validatedConfig match {
+    case Return(cfg) => cfg
+    case Throw(ex) => fail(s"expected config to parse successfully, but got $ex")
+  }
+
+  test("baseDtab on linker") {
+    assert(baseConfig.baseDtab === Some(Dtab.read("/foo => /bar ;")))
+  }
+
+  test("simple http router") {
+    val baseRouter = baseConfig.routers.get.head.asInstanceOf[HttpRouterConfig]
+    assert(baseRouter.httpUriInDst === Some(true))
+    assert(baseRouter.label == None) // label should default to protocol
+
+    /*
+    TODO: assertions on Stack.Params!!
```

Comment:
Oh man looks like the tests in this file don't compile and aren't being run.  Investigating now, but I think this is actually a legacy file and should be removed.